test/prod/ProdTest.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import {SafeTransferLib, ERC20} from "@rari-capital/solmate/src/utils/SafeTransferLib.sol";
+
+import {PercentageMath} from "@morpho-dao/morpho-utils/math/PercentageMath.sol";
+import {Math} from "@morpho-dao/morpho-utils/math/Math.sol";
+import {WadRayMath} from "@morpho-dao/morpho-utils/math/WadRayMath.sol";
+
+import {BaseConfig} from "config/BaseConfig.sol";
+import "@forge-std/console.sol";
+import "@forge-std/Test.sol";
+
+contract ProdTest is Test, BaseConfig {
+    // Show block number for reproducibility.
+    function testShowBlockNumber() public view {
+        console.log("Testing at block", block.number);
+    }
+
+    // Needed because AAVE packs the balance struct.
+    function deal(
```

Comment:
why was it needed if we can just give type(uint104).max ?

```solidity
         // Only 104 bits are used for the balance in the AAVE balance slot
         deal(aave, address(this), type(uint104).max);
```

Reply:
We can't do `type(uint104).max` only because it would revert. This is a recent forge upgrade that prevents doing `deal` when the target slot is a struct